src/cloud/social/provider.ts

```diff
@@ -363,26 +356,17 @@ export class CloudSocialProvider {
   // social2
   ////
 
-  // Returns a new invite code for the specified server.
-  // Rejects if the user is not logged in as an admin or there
-  // is any error executing the command.
-  // TODO: typings for invite codes
-  public inviteUser = (clientId: string): Promise<Object> => {
+  // Returns the invite code for the specified server.
+  public inviteUser = (host: string): Promise<Object> => {
```

Comment:
OK. There's probably no advantage to exporting it, right, this should just be an opaque object to the rest of the system.

Since it's intended to be re-consumed by other methods in this class I suggest casting it when you return, i.e.
```typescript
return Promise.resolve(<Invite>{
  host: invite.host,
  user: invite.user,
  key: invite.key
});
```